pkg/apis/v1/nodepool.go

```diff
@@ -156,7 +156,7 @@ func (l Limits) ExceededBy(resources v1.ResourceList) error {
 type NodeClaimTemplate struct {
 	ObjectMeta `json:"metadata,omitempty"`
 	// +required
-	Spec NodeClaimSpec `json:"spec"`
+	Spec NodeClaimTemplateSpec `json:"spec"`
```

Comment:
Why did this change? Why not just use the same NodeClaimSpec? What's the restriction on us doing that here?

Reply:
Removed from this PR